README.md

```diff
@@ -15,21 +15,38 @@ the Firebase iOS SDK. If you're interested in using the Firebase iOS SDK, start
 ## Context
 
 This repo contains a fully functional development environment for FirebaseCore,
-FirebaseAuth, FirebaseDatabase, FirebaseMessaging, and FirebaseStorage. By
+FirebaseAuth, FirebaseDatabase, FirebaseFirestore, FirebaseMessaging, and
+FirebaseStorage. By
```

Comment:
Nit: spacing here? The next line should be brought up to this line.

Reply:
Looks right when rendered at https://github.com/firebase/firebase-ios-sdk/blob/pb-future/README.md.  I thought it might be better to keep more of the blame intact in the source.

Thanks for the review.  I'm planning to present in Tuesday's leads meeting and hope to merge after that.

Reply:
SGTM.

Reply:
This file is also viewed in text editors; let's keep things looking crisp.